test/index_tests.js

```diff
@@ -62,6 +62,8 @@ describe('index.js', () => {
 
   beforeAll(done => {
     const app = express();
+
+    //Default args supplied to init. Only tag is modified
```

Comment:
Not sure that this comment is clear. Which tag is modified here? Looks like call of `init` function remaned the same.

Reply:
Finally used another test dataset 😄 Forgot to remove that comment, already done.
https://github.com/mpashkovskiy/express-oas-generator/pull/62#discussion_r464090623